pkg/task/task.go

```diff
@@ -861,22 +866,62 @@ func (m *taskExecutor) getOEV1alpha1SP() (err error) {
 
 // getExtnV1B1Deployment will get the Deployment as specified in the RunTask
 func (m *taskExecutor) getExtnV1B1Deployment() (err error) {
-	deploy, err := m.getK8sClient().GetExtnV1B1DeploymentAsRaw(m.getTaskObjectName())
+	dclient := deploy_extnv1beta1.KubeClient(
+		deploy_extnv1beta1.WithNamespace(m.getTaskRunNamespace()),
+		deploy_extnv1beta1.WithClientset(m.getK8sClient().GetKCS()))
+	d, err := dclient.Get(m.getTaskObjectName())
```

Comment:
This is a late comment.
Think about these possibilities as well:
```go
d, err := dclient.GetRaw(m.getTaskObjectNames())
```

Reply:
done